### PR TITLE
Allow Array{UInt8} to be passed in for content in RestXMLService

### PR DESCRIPTION
**Pull request by mattBrzezinski**
Resolves: #216

### CODE REVIEW

**Comment by mjram0s** [comment]
Is there an issue attached to this?

**Review by mjram0s — APPROVED** [review]

**Comment by mattBrzezinski** [comment]
> Is there an issue attached to this?

Forgot to link to it. Updated the top post with a link for it.

**Comment by mattBrzezinski** [comment]
The alternative solution to this is;

```julia
content=String(args, "body", "")
```

I think this is a more clean solution.

**Comment by mjram0s** [comment]
> The alternative solution to this is;
> 
> ```julia
> content=String(args, "body", "")
> ```
> 
> I think this is a more clean solution.

I agree

File: src/AWS.jl [hunk 1]
```diff
@@ -155,7 +155,7 @@ Base.@kwdef mutable struct Request
     request_method::String
 
     headers::AbstractDict{String, String}=LittleDict{String, String}()
-    content::String=""
+    content::Union{String, Array{UInt8}}=""
```
Comment (iamed2) [inline src/AWS.jl:158]:
```suggestion
    content::Union{String, Vector{UInt8}}=""
```

I don't think it's necessary to support nd arrays here, and keeping the type concrete will help the compiler.

**Review by iamed2 — APPROVED** [review]

**Pull request merged**